src/Features/CSharp/Portable/Completion/KeywordRecommenders/ParamsKeywordRecommender.cs

```diff
@@ -15,6 +15,10 @@ public ParamsKeywordRecommender()
         }
 
         protected override bool IsValidContext(int position, CSharpSyntaxContext context, CancellationToken cancellationToken)
-            => context.SyntaxTree.IsParamsModifierContext(context.Position, context.LeftToken);
+        {
+            var syntaxTree = context.SyntaxTree;
+            return syntaxTree.IsParamsModifierContext(context.Position, context.LeftToken) ||
+                syntaxTree.IsPossibleLambdaParameterModifierContext(position, context.LeftToken, cancellationToken);
```

Comment:
We should fix up IsParamsModifierContext to support lambdas.

Reply:
Right, that's better, thanks. Updated: 4c2f831fdf74362489629e3ef36f5d09a5bfd804